pkg/server/status_local_file_retrieval.go

```diff
@@ -59,6 +62,24 @@ func profileLocal(
 			return nil, err
 		}
 		return &serverpb.JSONResponse{Data: buf.Bytes()}, nil
+	case serverpb.ProfileRequest_GOROUTINE:
+		p := pprof.Lookup("goroutine")
+		if p == nil {
+			return nil, status.Error(codes.InvalidArgument, "unable to find goroutine profile")
+		}
+		var debug int
+		buf := bytes.NewBuffer(nil)
```

Comment:
totally up to you, just wanted to point out that `var buf bytes.NewBuffer` works too.